packages/app/src/pages/landelijk/verpleeghuiszorg.tsx

```diff
@@ -56,7 +54,7 @@ const NursingHomeCare = (props: StaticProps<typeof getStaticProps>) => {
   const infectedLocationsText = siteText.verpleeghuis_besmette_locaties;
   const positiveTestedPeopleText =
     siteText.verpleeghuis_positief_geteste_personen;
-  const locationDeaths = siteText.verpleeghuis_oversterfte;
+  const deceased = siteText.verpleeghuis_oversterfte;
```

Comment:
isn't `deceased` is a bit too vague?
```suggestion
  const deceasedText = siteText.verpleeghuis_oversterfte;
```